test/NetCore.Test/ExceptionTests.cs

```diff
@@ -29,5 +30,16 @@ public ExceptionTests()
             var exception = await Assert.ThrowsAsync<InvalidOperationException>(() => grain.ThrowsInvalidOperationException());
             Assert.Equal("Test exception", exception.Message);
         }
+
+        public void Dispose()
+        {
+            var cancel = new CancellationTokenSource();
+            cancel.Cancel();
+            this.silo.StopAsync(cancel.Token).GetAwaiter().GetResult();
+            this.silo?.Dispose();
```

Comment:
It's odd that you are using the null propagator here, but not above. Shouldn't it use it everywhere? Same with the `client.Abort()` call

Reply:
ReSharper did it when generating the method. I'll remove them

Reply:
Or I'll add them to the other locations. It's possible that this class would initialize a silo and fail to initialize a client. In that case, Dispose won't be called and the silo will live on in memory.

Reply:
I meant you probably want to add them everywhere, since it might fail to start the silo or the client and never assign those fields